pwndbg/vmmap.py

```diff
@@ -243,6 +261,7 @@ def coredump_maps():
     vsyscall_page = pages[-1]
     if vsyscall_page.start > 0xFFFFFFFFFF000000 and vsyscall_page.flags & 1:
         vsyscall_page.objfile = "[vsyscall]"
+        vsyscall_page.offset = 0
```

Comment:
Not really needed since we hardcode offset above to 0, but idk. Those pages *really* have it as 0